src/orderbook.rs

```diff
@@ -83,7 +83,15 @@ pub enum OrderState {
     REJECT,
 }
 
-// Create Order
+/// Used for order upate events after an order has been successfully matches
+/// and all parties need to be notified
+#[derive(Debug, Clone, Encode, Decode, PartialEq)]
+pub struct OrderUpdate {
+    pub order: Order,
+    pub balance: BalancesData, // contains the updated balance information of an account
```

Comment:
In the case of multiple orders of a given being filled,  BalancesData would be redundant. I would prefer the balanceUpdate to be independent and communicate the final result of a trade. even though there are multiple orderUpdate events for that trade

Reply:
So instead of an OrderUpdate you'd rather just give an `Order` as an Update (including it's status) and another call with a BalanceUpdate for a given account?

Reply:
Removed Balance Data from order update.